mora/service/facet.py

```diff
@@ -22,23 +22,24 @@
 
 '''
 
-import locale
 import itertools
+import locale
 import uuid
 
 import flask
 import werkzeug
 
-from .. import util
 from . import common
+from .keys import ADDRESS_TYPE, JOB_FUNCTION, ORG_UNIT_TYPE
+from .. import util
 
 blueprint = flask.Blueprint('facet', __name__, static_url_path='',
                             url_prefix='/service')
 
 FACETS = {
-    'address': 'Adressetype',
-    'job-function': 'Stillingsbetegnelse',
-    'ou': 'Enhedstype',
+    ADDRESS_TYPE: 'Adressetype',
+    JOB_FUNCTION: 'Stillingsbetegnelse',
+    ORG_UNIT_TYPE: 'Enhedstype',
```

Comment:
We should use keys for the strings at the right as well. Also, note that you actually change the URLs to use underscore — is this intentional? (I'm not a fan of underscores in URLs — if so, we should consider converting them to query parameters.)

Reply:
My idea here, is to have the names be competely identical between getting them from the the facet endpoint to adding them as a parameter to a request. E.g. the facet from 'job_function' goes in the 'job_function' field - No confusion, no ambiguity.

Reply:
We'll let them be for now; we can drop the underscore later, and perhaps drop it later when everything works.